MetaMask/metamask-extension
# feat: Add Oasis Sapphire Mainnet and Testnet

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28168?quickstart=1)

## **Related issues**

Adds SVG icon for Oasis Sapphire Mainnet and Testnet similar to https://github.com/MetaMask/metamask-extension/pull/27841

Close https://github.com/MetaMask/metamask-extension/issues/28205

## **Manual testing steps**

1. Go to https://docs.oasis.io/dapp/sapphire/network/
2. Click "Add to MetaMask"
3. See that network icon is the new loaded SVG

## **Screenshots/Recordings**



### **Before**



https://github.com/user-attachments/assets/e01ae508-6239-42dc-8037-88ef9a227c9b

### **After**



https://github.com/user-attachments/assets/0ad73b86-b2e8-4781-95ad-a2616b0a1b6d

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
